views/components/Sidebar/SidebarFooter.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import {
+  Typography,
+  Card,
+  Avatar,
+  CardHeader,
+  CardContent,
+  CardActions,
+  Button,
+} from '@mui/material'
+import { Link } from 'react-router-dom'
+import { routes } from '../../routes/routes'
+import { FontSize14InRem } from '../../../constants'
+import './SidebarFooter.css'
+
+const SidebarFooter = ({ user, onLogout }) => {
+  return (
+    <Card
+      sx={{ border: 0, boxShadow: 0, color: 'text.primary', height: '236px' }}
```

Comment:
do we need to set this height?

Reply:
[377af01](https://github.com/AMP-SCZ/dpdash/pull/458/commits/377af0123d164798059bf0dc152683f783a6b3f8)